metals/src/main/scala/scala/meta/internal/metals/doctor/Doctor.scala

```diff
@@ -115,7 +120,10 @@ final class Doctor(
         case DoctorFormat.Html => buildTargetsHtml()
       }
       val params = clientCommand.toExecuteCommandParams(output)
-      languageClient.metalsExecuteClientCommand(params)
+      val shouldDisplayForVsCode = (clientConfig.isVscode && isVisible.get())
+      if (shouldDisplayForVsCode || !clientConfig.isVscode) {
```

Comment:
Check visible status only for vscode - this gives other editors some time to adjust.